meson.build

```diff
@@ -56,15 +55,77 @@ else
     'contrib/htslib/cram/rANS_static.c',
     'contrib/htslib/cram/open_trace_file.c',
     'contrib/htslib/multipart.c',
-]
+    )
+tabixpp_inc = include_directories('vcflib/tabixpp')
+tabixpp_src = files('vcflib/tabixpp/tabix.cpp')
+vcflib_inc = include_directories(
+    'vcflib/src',
+    'vcflib/smithwaterman',
+    'vcflib/multichoose',
+    'vcflib/filevercmp',
+    'vcflib/fastahack',
+    )
+vcflib_src = files(
+    'vcflib/src/Variant.cpp',
+    'vcflib/smithwaterman/SmithWatermanGotoh.cpp',
+    'vcflib/smithwaterman/disorder.cpp',
+    'vcflib/smithwaterman/Repeats.cpp',
+    'vcflib/smithwaterman/LeftAlign.cpp',
+    'vcflib/smithwaterman/IndelAllele.cpp',
+    'vcflib/fastahack/Fasta.cpp',
+    'vcflib/smithwaterman/LeftAlign.cpp',
+    'vcflib/smithwaterman/IndelAllele.cpp',
+    'vcflib/src/split.cpp',
+    )
+seqlib_inc = include_directories(
+    'contrib',
+    'contrib/SeqLib',
+    )
+seqlib_src = files(
+    'contrib/SeqLib/src/BamReader.cpp',
+    'contrib/SeqLib/src/BamRecord.cpp',
+    'contrib/SeqLib/src/BamHeader.cpp',
+    'contrib/SeqLib/src/BamWriter.cpp',
+    'contrib/SeqLib/src/GenomicRegion.cpp',
+    'contrib/SeqLib/src/ssw_cpp.cpp',
+    'contrib/SeqLib/src/ssw.c',
+    )
+
+if not htslib_dep.found()
+    htslib_lib = static_library('custom_htslib',
+                                htslib_src,
+                                include_directories : htslib_inc)
+    htslib_dep = declare_dependency(link_with : htslib_lib,
+                                    include_directories : htslib_inc)
+endif
+if not tabixpp_dep.found()
+    tabixpp_lib = static_library('custom_tabixpp',
+                                 tabixpp_src,
+                                 include_directories : tabixpp_inc)
+    tabixpp_dep = declare_dependency(link_with : tabixpp_lib,
+                                     include_directories : tabixpp_inc)
+endif
+if not vcflib_dep.found()
+    vcflib_lib = static_library('custom_vcflib',
+                                vcflib_src,
+                                include_directories : vcflib_inc)
```

Comment:
vcflib uses tabix, so this is supposed to be include_directories : [vcflib_inc, tabixpp_inc]